clients/src/main/java/org/apache/kafka/common/MessageFormatter.java

```diff
@@ -34,33 +33,21 @@
  */
 public interface MessageFormatter extends Configurable, Closeable {
 
-    /**
-     * Initialises the MessageFormatter
-     * @param props Properties to configure the formatter
-     * @deprecated Use {@link #configure(Map)} instead, this method is for backward compatibility with the older Formatter interface
-     */
-    @Deprecated
```

Comment:
If we remove this one, we could also remove the `kafka.common.MessageFormatter` trait in the core module.

Reply:
Good point, done.